pkg/solana/chain.go

```diff
@@ -557,7 +557,7 @@ func (c *chain) sendTx(ctx context.Context, from, to string, amount *big.Int, ba
 				toKey,
 			).Build(),
 		},
-		blockhash.Value.Blockhash,
+		blockhash.Value.Blockhash, // Will be override if needed within sendWithRetry txm function.
```

Comment:
nit: just a grammar thing but we also probably don't need to add the internal TXM details here since this is outside of the TXM
```suggestion
		blockhash.Value.Blockhash, // Will be overridden by the TXM if needed
```

Reply:
thanks, I'll remove it